internal/drivers/basic/ratelimit.go

```diff
@@ -50,16 +50,16 @@ var (
 		keppel.AnycastBlobBytePullAction: {"KEPPEL_RATELIMIT_ANYCAST_BLOB_PULL_BYTES", "KEPPEL_BURST_ANYCAST_BLOB_PULL_BYTES"},
 	}
 	valueRx          = regexp.MustCompile(`^\s*([0-9]+)\s*[Br]/([smh])\s*$`)
-	rateConstructors = map[string]func(int) throttled.Rate{
-		"s": throttled.PerSec,
-		"m": throttled.PerMin,
-		"h": throttled.PerHour,
+	rateConstructors = map[string]func(int) redis_rate.Limit{
```

Comment:
Since this now constructs `Limit` instances, it should be called `limitConstructors`.